src/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -590,6 +590,61 @@ public static class MsalError
         /// </summary>
         public const string ClientCredentialAuthenticationTypesAreMutuallyExclusive = "Client_Credential_Authentication_Types_Are_Mutually_Exclusive";
 
+        #region InvalidGrant suberrors
+        /// <summary>
+        /// Condition can be resolved by user interaction during the interactive authentication flow.
```

Comment:
Condition? not issue?

Reply:
This is an internal comment now, I will fix it nontheless.